nativelib/src/main/scala/scala/scalanative/runtime/package.scala

```diff
@@ -54,6 +54,6 @@ package object runtime {
   def init(argc: Int, argv: Ptr[Ptr[Byte]]): ObjectArray = {
     GC.init()
     null
-  }
+  }  
```

Comment:
I can't see the full outdated diff. Scalafmt should complain about trailing whitespace so I am curious to know what caused this.